custom_dc/custom_data.md

```diff
@@ -277,7 +279,7 @@ If you need to troubleshoot custom data, it is helpful to inspect the contents o
 To do so, from a terminal window, open the database:
 
 <pre>  
-sqlite3 website/custom_dc/<var>CUSTOM_DATA_DIRECTORY</var>/datacommons/datacommons.db
+sqlite3 <var>WEBSITE_ROOT</var>/<var>OUTPUT_DIRECTORY</var>/datacommons/datacommons.db
```

Comment:
The output directory need not be under the website root. So the path can just be `<var>OUTPUT_DIRECTORY</var>/datacommons/datacommons.db`

Reply:
oh, OK, thanks